public/partials/fixed_sessions_map.html

```diff
@@ -55,3 +54,5 @@ <h4>Heat Legend</h4>
     </div>
   </div>
 </div>
+
+<link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/daterangepicker/daterangepicker.css" />
```

Comment:
Maybe we could have this in the layout that is used for both mobile and fixed map?